### PR TITLE
Updated SOW

### PR DESCRIPTION
**Pull request by tthorleifson**
Added several desired SOW requirements to sections 1. and 3., and a few other minor revisions. Please review and revise/comment, as needed.

### CODE REVIEW

File: SOW.md [hunk 1]
```diff
@@ -20,24 +20,25 @@ a permissive license allowing commercial uses.
...
-	* Adding evaporation and infiltration models.
+	* Adding evaporation and infiltration models, with configurable input parameters.
 
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Other modifications may also be required, based on the input from G2-IS during the course of this project.
 
 2. Performance improvement to achieve runtimes in the order of minutes per rupture point. Possible approaches include:
 
-	* Taking advantage of Clawpack's multi-core parallelism (OpenMP).
-	* Experimenting with different settings for adaptive mesh refinement.
+	* Taking advantage of Clawpack's multi-core parallelism (OpenMP), including Graphical Processing Unit (GPU) processing.
```
Comment (labarba) [inline SOW.md]: @piyueh Please review: Geoclaw doesn't have GPU capacity, but there is CuClaw now. The Clawpack version used by Geoclaw is many versions behind. Any estimation of the chances to get it working with the newer versions?

File: SOW.md [hunk 2]
```diff
@@ -46,10 +47,14 @@ All software deliverables will open source, under a permissive license.
 	* Create a simple way to launch simulations, without needing to re-compile for different initial conditions.
 	* Simplify the way to configure the various simulation parameters.
 	* Integrate with Jupyter Notebooks for seamless workflows and reports.
-	* Integrate with GIS software, as needed.
-	* Study the use of Amazon AWS or other cloud services.
+	* Design for implementation via REST web service.
+	* Integrate with Esri GIS software, as needed.
```
Comment (labarba) [inline SOW.md:51]: @tthorleifson As you know, we don't have access to the proprietary GIS tools from ESRI. Do these use open formats, so we could develop with an open-source GIS alternative? Or do you have chance of getting us a trial license for a few months?

File: SOW.md (hunk 1 above)
Comment (labarba) [reply SOW.md]: Agreed by email that GPU is postponed to potential continued work. Will edit this separately.

Comment (tthorleifson) [reply SOW.md]: Agreed.

File: SOW.md (hunk 2 above)
Comment (tthorleifson) [reply SOW.md:51]: Working on it! :-)

**Pull request merged**